### PR TITLE
Update icons

### PR DESCRIPTION
**Pull request by phallobst**

### CODE REVIEW

File: img/f/foursquare.com.svg [hunk 1]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 67 81"><path fill="currentColor" d="M.703 0h24.764v4.064H4.814v11.379h18.358v4.064H4.814v16.449H.704V0zM0 68.043l4.018-.908c.717 4.972 4.157 7.938 9.754 7.938 5.018 0 8.99-2.342 8.99-6.169 0-2.723-2.007-5.019-9.852-7.36C4.018 59.01 1.052 55.757 1.052 50.98c0-6.17 4.972-9.465 12.194-9.465 8.077 0 11.667 4.11 12.91 9.753l-4.018.908C21.132 47.54 17.98 45.58 13.1 45.58c-4.59 0-7.84 1.723-7.84 5.019 0 2.677 2.104 4.735 9.372 7.03 8.655 2.677 12.384 5.596 12.384 11.141 0 6.886-5.787 10.47-13.148 10.47C6.17 79.232.908 75.214 0 68.044zm31.128-7.556c0-10.662 6.979-18.982 17.692-18.982 10.661 0 17.547 8.367 17.547 18.982 0 5.307-1.77 9.898-4.735 13.245a122.024 122.024 0 0 1 4.209 4.4L62.875 81a192.912 192.912 0 0 0-4.4-4.544c-2.723 1.769-5.977 2.775-9.655 2.775-10.331 0-17.692-7.938-17.692-18.744zm24.237 12.91a105.44 105.44 0 0 0-4.208-3.874l2.868-2.914a123.314 123.314 0 0 1 4.492 4.11c2.151-2.578 3.441-6.117 3.441-10.279 0-8.413-5.117-14.819-13.148-14.819-8.03 0-13.194 6.406-13.194 14.82 0 8.654 5.592 14.725 13.194 14.725 2.445 0 4.642-.624 6.555-1.769z"/></svg>
```
Comment (Carlgo11) [inline img/f/foursquare.com.svg]: There are two fill tags here :thinking:

File: img/n/noke.com.svg [hunk 2]
```diff
@@ -0,0 +1 @@
....cls-1{fill:#0071ce}.cls-2{fill:#939598}</style></defs><g id="g2246" transform="translate(-.015 19.052)"><path id="path1683" transform="translate(-9.98 -5.41)" d="M25.57 32h-1.4V20a6.4 6.4 0 1 0-12.79 0v12H10V20a7.8 7.8 0 1 1 15.59 0z" class="cls-2" style="fill:#939598"/><path id="path1685" transform="translate(-9.98 -5.41)" d="M36.55 32c-4.94 0-8.67-4.25-8.67-9.89 0-5.64 3.73-9.89 8.67-9.89 4.94 0 8.66 4.25 8.66 9.89 0 5.64-3.73 9.89-8.66 9.89zm0-18.38c-4.21 0-7.27 3.57-7.27 8.49s3.06 8.49 7.27 8.49c4.21 0 7.26-3.57 7.26-8.49s-3.06-8.49-7.26-8.49z" class="cls-2" style="fill:#939598"/><path id="path1687" transform="translate(-9.98 -5.41)" d="M78.29 22.32h.4v-.21c0-5.82-3.41-9.89-8.3-9.89-4.89 0-8.66 4.25-8.66 9.89 0 5.64 3.74 9.89 8.67 9.89 4 0 7-2.69 7.95-6.83h-1.46c-.91 3.3-3.32 5.43-6.49 5.43-4.14 0-7.16-3.47-7.25-8.28zm-7.89-8.7c3.73 0 6.42 3 6.83 7.3h-14c.44-4.29 3.32-7.3 7.17-7.3z" class="cls-1" style="fill:#0071ce"/><path id="polygon1689" class="cls-1" style="fill:#0071ce" d="M38.95 16.61V0h-1.4v26.59h1.4v-8.14l3.61-3.07 8.46 11.21h1.75l-9.14-12.11 9.12-7.74h-2.17z"/><path id="polygon1691" class="cls-1" style="fill:#0071ce" d="M67.13 3.33H56.75l-1.57 1.33h11.95z"/></g></svg>
```
Comment (Carlgo11) [inline img/n/noke.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 68.69 26.59"><path d="M15.57 26.59h-1.4v-12a6.4 6.4 0 1 0-12.79 0v12H0v-12a7.8 7.8 0 1 1 15.6 0zm10.98 0c-4.94 0-8.67-4.25-8.67-9.89 0-5.64 3.73-9.89 8.67-9.89 4.94 0 8.66 4.25 8.66 9.89 0 5.64-3.72 9.89-8.65 9.89zm0-18.38c-4.2 0-7.27 3.57-7.27 8.49s3.06 8.49 7.27 8.49c4.21 0 7.26-3.57 7.26-8.49s-3.06-8.49-7.26-8.49z" style="fill:#939598"/><path d="M68.3 16.91h.4v-.21c0-5.82-3.41-9.89-8.3-9.89-4.9 0-8.66 4.25-8.66 9.89 0 5.64 3.74 9.89 8.67 9.89 4 0 7-2.69 7.95-6.83H66.9c-.91 3.3-3.32 5.43-6.5 5.43-4.13 0-7.15-3.47-7.24-8.28zm-7.9-8.7c3.74 0 6.43 3 6.84 7.3h-14c.43-4.29 3.31-7.3 7.16-7.3zm-21.46 8.4V0h-1.4v26.59h1.4v-8.14l3.6-3.07 8.47 11.21h1.75l-9.14-12.11 9.12-7.74h-2.18zM67.11 3.33H56.73l-1.57 1.33h11.95z" style="fill:#0071ce"/></svg>
```

File: img/n/nubank.com.br.svg [hunk 3]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" width="43.5" height="43.5" viewBox="0 0 57.34 58"><path style="fill:#820ad1;fill-opacity:1" class="sc-pQEbo yAvEL" d="M10.47 3.05A9.62 9.62 0 0 1 17.6 0c5.56 0 9.25 4.07 9.96 10.16.23 1.97.23 4.76.23 8v13.1h-7.66V22.3s-.02-7.66-.06-9.1c-.21-6.23-3.9-10.14-9.6-10.15C8.75 4.88 7.83 7.1 7.69 10.5c-.02.47-.01 2.15 0 4.37v16.39H.03V17.49l-.01-1.43c-.02-.96-.04-1.93.01-2.89.1-1.6.37-3.17 1.11-4.63a10.12 10.12 0 0 1 9.33-5.49zM57.97 18.83c.05-.96.03-1.93.01-2.9l-.01-1.42V.74H50.3v16.39c.01 2.22.02 3.9 0 4.37-.14 3.4-1.06 5.62-2.78 7.44-5.7 0-9.39-3.92-9.6-10.16-.04-1.43-.06-4.98-.06-9.1V.74h-7.65v13.1a79 79 0 0 0 .22 8c.7 6.1 4.4 10.16 9.96 10.16 2.86 0 5.3-1.13 7.13-3.05h.39c3.73 0 7.23-2.15 8.94-5.5a11.34 11.34 0 0 0 1.1-4.62z" transform="translate(-.33 13)"/></svg>
```
Comment (Carlgo11) [inline img/n/nubank.com.br.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 58 32"><path fill="#820ad1" d="M10.47 3.05A9.62 9.62 0 0 1 17.6 0c5.56 0 9.25 4.07 9.96 10.16.23 1.97.23 4.76.23 8v13.1h-7.66V22.3s-.02-7.66-.06-9.1c-.2-6.23-3.9-10.14-9.6-10.15C8.75 4.88 7.83 7.1 7.7 10.5c-.02.47 0 2.15 0 4.37v16.4H.03V16.05c-.03-.96-.05-1.93 0-2.89.1-1.6.37-3.17 1.11-4.63a10.12 10.12 0 0 1 9.33-5.49zm47.5 15.78c.05-.96.03-1.93.01-2.9V.74H50.3v16.4c.01 2.21.02 3.9 0 4.36-.14 3.4-1.06 5.62-2.78 7.44-5.7 0-9.39-3.92-9.6-10.16-.04-1.43-.06-4.98-.06-9.1V.74h-7.65v13.1a79 79 0 0 0 .22 8c.7 6.1 4.4 10.16 9.96 10.16 2.86 0 5.3-1.13 7.13-3.05h.4c3.72 0 7.22-2.15 8.93-5.5a11.34 11.34 0 0 0 1.1-4.62z"/></svg>
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: img/t/target.com.svg [hunk 4]
```diff
@@ -1 +1 @@
-<svg xmlns="http://www.w3.org/2000/svg" width="21.999" height="21.999"><path d="M11 0c6.074 0 10.999 4.925 10.999 11 0 6.074-4.925 10.999-11 10.999-6.074 0-10.999-4.925-10.999-11C0 4.925 4.925 0 11 0zm0 18.173a7.174 7.174 0 10-.001-14.347 7.174 7.174 0 000 14.347zm0-3.653a3.52 3.52 0 110-7.04 3.52 3.52 0 010 7.04z" fill-rule="evenodd" fill="#c00"/></svg>
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 -4.5 .5 9"><circle r="3.7" fill="none" stroke="#c00" stroke-width="1.6"/><circle r="1.4" fill="#c00"/></svg>
```
Comment (phallobst) [inline img/t/target.com.svg]: The `viewBox` for this one is broken.

Comment (Carlgo11) [reply img/t/target.com.svg]: Nice catch! :+1:

**Review by Carlgo11 — APPROVED** [review]

File: img/f/foursquare.com.svg [hunk 5]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 67 81"><path d="M.7 0h24.77v4.06H4.8v11.38h18.36v4.07H4.81v16.45H.71V0zM0 68.04l4.02-.9c.72 4.97 4.16 7.93 9.75 7.93 5.02 0 9-2.34 9-6.17 0-2.72-2.01-5.01-9.86-7.36-8.9-2.53-11.86-5.78-11.86-10.56 0-6.17 4.97-9.46 12.2-9.46 8.07 0 11.66 4.1 12.9 9.75l-4.01.9c-1-4.63-4.16-6.59-9.04-6.59-4.59 0-7.84 1.72-7.84 5.02 0 2.68 2.1 4.73 9.37 7.03 8.66 2.68 12.39 5.6 12.39 11.14 0 6.89-5.8 10.47-13.15 10.47C6.17 79.24.9 75.21 0 68.04zm31.13-7.55c0-10.66 6.98-18.98 17.69-18.98 10.66 0 17.55 8.36 17.55 18.98 0 5.3-1.77 9.9-4.74 13.24a122.02 122.02 0 0 1 4.21 4.4L62.87 81a192.91 192.91 0 0 0-4.4-4.54 17.48 17.48 0 0 1-9.65 2.77c-10.33 0-17.7-7.94-17.7-18.74zm24.23 12.9a105.44 105.44 0 0 0-4.2-3.87l2.86-2.91a123.31 123.31 0 0 1 4.5 4.1c2.15-2.57 3.44-6.1 3.44-10.27 0-8.41-5.12-14.82-13.15-14.82s-13.2 6.4-13.2 14.82c0 8.66 5.6 14.73 13.2 14.73 2.44 0 4.64-.63 6.55-1.77z"/></svg>
```
Comment (ApCoder123) [inline img/f/foursquare.com.svg:1]: I'm not 100% sure whether this logo should be updated - the logo for the company seems to be the "FSQ" logo, however for the specific "City Guide" app, which the entry refers to, the current logo still seems to be in use. 
See:
- https://apps.apple.com/gb/app/foursquare-city-guide/id306934924
- https://foursquare.com/city-guide (favicon)
- From the "Products and Solutions" menu:
image

Comment (phallobst) [reply img/f/foursquare.com.svg:1]: https://foursquare.com/article/a-new-look-and-feel-introducing-foursquare-everywhere/
I'm afraid they can't even keep up with themselves... :smile:

**Review by ApCoder123 — APPROVED** [review]

**Pull request merged**